modules/docker/Makefile

```diff
@@ -71,6 +71,81 @@ docker/fatmanifest/build: docker/fatmanifest/config
 	$(DOCKER) --config $(DOCKER_CFG_FOLDER) manifest push --purge $(DOCKER_IMAGE_LATEST)
 	rm -rf $(DOCKER_CFG_FOLDER)
 
+# Builds a "fat" (multi-platform) Docker manifest.
+# Target docker/registry/login should be called first.
+#
+# Variables:
+#   DOCKER_IMAGE_NAME
+#     The name of the Docker manifest.  This is often of the
+#     form "registry/image:tag".
+#   DOCKER_PLATFORMS
+#     The platforms to include in the manifest.  Default:
+#     "amd64 ppc64le s390x".
+.PHONY: docker/arch/manifest/build
+## Builds a "fat" (multi-platform) Docker manifest
+docker/arch/manifest/build: docker/fatmanifest/config
```

Comment:
If this is just a copy/paste of the `docker/manifest/build` target, I _though_ we could just something like
```Make
docker/arch/manifest/build: docker/fatmanifest/build
```

So that even if the `docker/fatmanifest/build` target is deprecating we can still share the code if/until it's fully deprecated?

Reply:
or maybe it's the other way. around... the old deprecating target combines both our new targets?